src/System.Security.Cryptography.Xml/tests/TestHelpers.cs

```diff
@@ -2,10 +2,28 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
 using System.IO;
+using System.Text;
+using System.Xml;
 
 namespace System.Security.Cryptography.Xml.Tests
 {
+    public class TempFile : IDisposable
```

Comment:
This already exists in the common test code:

https://github.com/tintoy/corefx/blob/feature/xml-crypto/test-files-base-class/src/System.Security.Cryptography.Xml/tests/System.Security.Cryptography.Xml.Tests.csproj#L49